stable/kommander/values.yaml

```
@@ -201,6 +201,8 @@ kubecost:
       thanos:
         enabled: true
         queryService: http://kommander-kubecost-thanos-query-http.kommander:10902
+        # The offset to apply to all thanos queries in order to achieve syncronization on all cluster block stores
```

perhaps something a little more human friendly?
```suggestion
        # The wait time before Kommander begins querying cost data for all attached clusters
```

---

sgtm, I lifted the comment directly from kubecost but yeah not super clear what it's referring to :p